nemo/collections/nlp/modules/common/tokenizer_utils.py

```diff
@@ -49,34 +49,25 @@ def get_tokenizer(
         special_tokens: dict of special tokens
         vocab_file: path to vocab file
     """
-    full_huggingface_pretrained_model_list = get_huggingface_pretrained_lm_models_list(include_external=True)
```

Comment:
same question, why removing all the checks?

Reply:
updated